.github/workflows/eventarc-audit-storage.yaml

```diff
@@ -0,0 +1,45 @@
+name: eventarc-audit-storage
+on:
+  push:
+    branches:
+    - main
+    paths:
+    - 'eventarc/audit-storage/**'
+  pull_request:
+    paths:
+    - 'eventarc/audit-storage/**'
+  pull_request_target:
+    types: [labeled]
+  schedule:
+  - cron:  '0 2 * * *'
+jobs:
+  test:
+    if: ${{ github.event.action != 'labeled' || github.event.label.name == 'actions:force-run' }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    steps:
+    - uses: 'google-github-actions/auth@v0.3.1'
+      with:
+        workload_identity_provider: 'projects/1046198160504/locations/global/workloadIdentityPools/github-actions-pool/providers/github-actions-provider'
+        service_account: 'kokoro-system-test@long-door-651.iam.gserviceaccount.com'
+        create_credentials_file: 'true'
+        access_token_lifetime: 600s
```

Comment:
The default behavior does not create an access token (you need to use `token_format: access_token` to generate an access token or `token_format: id_token` for OIDC tokens.